integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java

```diff
@@ -43,7 +43,12 @@ public MicroserviceDeployDefinition getMicroserviceDeployDefinition() {
   @Override
   protected String[] createCmds() {
     // must set jar at the end of the cmds
-    return new String[] {"java", "-jar"};
+    return new String[] {"java", "-jar",
+        //add a property definition to enable quick search in taskmgr
+        "-DCombDeploy=true",
+        //remote debug, default disabled
+        //"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=0.0.0.0:" + RandomUtils.nextInt(60000, 61000)
```

Comment:
commented out code. we can make it a config in Deploy like heap memory config.

Reply:
removed